INFO     root:conftest.py:125 Starting test: test_digest_auth_login_scenarios[admin-wrong].
INFO     WDM:logger.py:11 xdist worker is: gw2
INFO     WDM:logger.py:11 ====== WebDriver manager ======
INFO     WDM:logger.py:11 Get LATEST chromedriver version for google-chrome
INFO     WDM:logger.py:11 Get LATEST chromedriver version for google-chrome
INFO     WDM:logger.py:11 Driver [/home/runner/.wdm/gw2/drivers/chromedriver/linux64/142.0.7444.59/chromedriver-linux64/chromedriver] found in cache
INFO     root:conftest.py:148 Started recording: tests_recordings/gw2/test_digest_auth_login_scenarios[admin-wrong]_20251029_125415/test_digest_auth_login_scenarios[admin-wrong]_20251029_125415.mp4
INFO     root:base_page.py:100 Navigating to URL: https://the-internet.herokuapp.com/.
INFO     root:base_page.py:102 Navigation completed.
INFO     root:base_page.py:82 Waiting for page to load with indicator '('css selector', 'h1.heading')' for 10s.
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
INFO     root:base_page.py:87 Page loaded successfully with indicator '('css selector', 'h1.heading')'.
INFO     root:page_manager.py:61 Navigating to Digest Authentication page
INFO     root:base_page.py:100 Navigating to URL: https://admin:wrong@the-internet.herokuapp.com/digest_auth.
INFO     root:base_page.py:102 Navigation completed.
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
ERROR    root:base_page.py:32 Timeout after 10s for locator ('css selector', 'div#content p'): Message: 
Stacktrace:
#0 0x55e9e344caea <unknown>
#1 0x55e9e2e98cdb <unknown>
#2 0x55e9e2eeb6c4 <unknown>
#3 0x55e9e2eeb901 <unknown>
#4 0x55e9e2f3a8b4 <unknown>
#5 0x55e9e2f37c87 <unknown>
#6 0x55e9e2eddaca <unknown>
#7 0x55e9e2ede7d1 <unknown>
#8 0x55e9e3413ab9 <unknown>
#9 0x55e9e3416a8c <unknown>
#10 0x55e9e33fcd49 <unknown>
#11 0x55e9e3417685 <unknown>
#12 0x55e9e33e46c3 <unknown>
#13 0x55e9e34397d8 <unknown>
#14 0x55e9e34399b3 <unknown>
#15 0x55e9e344ba83 <unknown>
#16 0x7f4a48e9caa4 <unknown>
#17 0x7f4a48f29c6c <unknown>

INFO     root:test_digest_auth_login_scenarios.py:26 Page source: <html><head></head><body></body></html>
INFO     root:conftest.py:152 Stopping video recording...
INFO     utils.video_recorder:video_recorder.py:95 Video created: tests_recordings/gw2/test_digest_auth_login_scenarios[admin-wrong]_20251029_125415/test_digest_auth_login_scenarios[admin-wrong]_20251029_125415.mp4
INFO     root:conftest.py:103 Quitting driver for browser: chrome.